lms/migrations/versions/8e203ad93a58_backfill_unique_teachers_for_existing_.py

```diff
@@ -0,0 +1,40 @@
+"""Backfill unique teachers for existing reports.
+
+Revision ID: 8e203ad93a58
+Revises: da01fe01c324
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+revision = "8e203ad93a58"
+down_revision = "da01fe01c324"
+
+
+def upgrade() -> None:
+    conn = op.get_bind()
+
+    conn.execute(
+        sa.text(
+            """
+        WITH teacher_counts as (
```

Comment:
The select side of the query can be test [in metabase](https://report.hypothes.is/question#eyJkYXRhc2V0X3F1ZXJ5Ijp7InR5cGUiOiJuYXRpdmUiLCJuYXRpdmUiOnsicXVlcnkiOiJXSVRIIHRlYWNoZXJfY291bnRzIGFzIChcblxuU0VMRUNUIFxuICAgIG9yZ2FuaXphdGlvbl91c2FnZV9yZXBvcnQuaWQsICBjb3VudChkaXN0aW5jdCBoX3VzZXJpZCApIEZJTFRFUihXSEVSRSBlbWFpbCA8PiAnPFNUVURFTlQ-JykgdW5pcXVlX3RlYWNoZXJzXG5GUk9NIFxuICAgIG9yZ2FuaXphdGlvbl91c2FnZV9yZXBvcnQsICAgXG4gICAganNvbmJfYXJyYXlfZWxlbWVudHMocmVwb3J0KSBBUyByZXBvcnRfcm93LFxuICAgIGpzb25iX3RvX3JlY29yZChyZXBvcnRfcm93KSBBUyB1c2FnZV91c2VyKGVtYWlsIHRleHQsIGhfdXNlcmlkIHRleHQpXG4gICAgZ3JvdXAgYnkgb3JnYW5pemF0aW9uX3VzYWdlX3JlcG9ydC5pZFxuKVxuXG4gICAgc2VsZWN0ICogZnJvbSB0ZWFjaGVyX2NvdW50c1xuXG5cbiIsInRlbXBsYXRlLXRhZ3MiOnt9fSwiZGF0YWJhc2UiOjV9LCJkaXNwbGF5IjoidGFibGUiLCJwYXJhbWV0ZXJzIjpbXSwidmlzdWFsaXphdGlvbl9zZXR0aW5ncyI6e319)